example/delaying-requests.jsonnet

```diff
@@ -0,0 +1,23 @@
+local test = import '../postman.libsonnet';
+
+/**
+ * This test suite demonstrates how to delay requests
+ */
+
+test.suite {
+  name: 'Delaying tests',
+  item:
```

Comment:
I think the syntax here will fail at compile time

Reply:
... it will not, simply did not know myself that postman accepted something else than a list in the spec for that field !